Scripts/travis.sh

```diff
@@ -108,7 +108,9 @@ elif [ "${TYPE}" == "EXAMPLE_PODS_SWIFT" ]; then
   cd gem/
   bundle install
   rake install
-  cd ..
+  cd ../Demo/EarlGreyExample
+  pod update
```

Comment:
The API was updated.  If it was an error with EG or the tests, then FTRSwift tests would have failed as well. I'd surmise it's failing here but not failing after a Pod Update is because it is using a stale pods version. We aren't guaranteed a clean build by travis if I'm right.